src/Servers/IIS/IIS/src/NativeMethods.cs

```diff
@@ -47,13 +47,13 @@ public enum REQUEST_NOTIFICATION_STATUS
         public delegate void PFN_REQUESTS_DRAINED_HANDLER(IntPtr pvRequestContext);
 
         [DllImport(AspNetCoreModuleDll)]
-        private static extern int http_post_completion(IntPtr pInProcessHandler, int cbBytes);
+        private static extern int http_post_completion(HandlerSafeHandle pInProcessHandler, int cbBytes);
```

Comment:
Shouldn't these be all typed as safe handle?

Reply:
They can, I just started with the handler.